### DISCUSSION TITLE
Custom widget with built-in session

### DISCUSSION THREAD

**Discussion by Maycon-Duarte** [elementor/elementor#23205] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When developing custom widgets seeking to create components that are within web accessibility standards, I found myself having to create duplicates of the same widget for different occasions, this is inefficient and tiring

### Describe the solution you'd like

If I could define a section inside my widget where in the editor the user can drag widgets inside as needed, having complete freedom to change the type of content, something very close to the behavior of an inner section but inside a widget

### Describe alternatives you've considered

For now I'm using the REPEATER controler to simulate sections of content and SWITCHER to change how the content is displayed

### Additional context

![image](https://github.com/elementor/elementor/assets/69111330/120ada4d-dc9f-466b-a948-5c9df53953dd)

The threads I found covering this subject were not entirely related to creating custom widgets

**Comment by m1ga** [comment]
there was some info that the nested functionality is only available for first party widgets at the moment: https://github.com/orgs/elementor/discussions/20732#discussioncomment-4393357
But yes, would be nice to use that in own custom widgets!
